code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -942,7 +942,12 @@ Pass a positive integer as an argument to override a bot's default speed.
 				ejectpai(usr)
 
 /mob/living/simple_animal/bot/update_icon_state()
-	icon_state = "[initial(icon_state)][get_bot_flag(bot_mode_flags, BOT_MODE_ON)]"
+	var/base_state
+	if(!isnull(base_icon_state))
```

Comment:
Invert this and also probably use a ternary, shouldn't `if (!x) else` for short bodies